apps/ledger-live-desktop/src/renderer/Default.jsx

```diff
@@ -254,6 +255,10 @@ export default function Default() {
                               render={(props: any) => <Asset {...props} />}
                             />
                             <Route path="/swap" render={props => <Swap2 {...props} />} />
+                            <Route
+                              path="/custom-image"
+                              render={(props: any) => <CustomImage {...props} />}
```

Comment:
Maybe you can get the type of `props` by exporting `Props` from `CustomImage` ?

Reply:
Actually this route is not needed haha I was using it for debugging before putting `CustomImage` in a Drawer.
Thanks for highlighting that 👍